lib/foods.js

```diff
@@ -4,93 +4,154 @@
 const herokuUrl = require("./heroku-url")
 const $ = require("jQuery")
 
-$('#newFoodForm').hide();
```

Comment:
Love the removal of the semicolons.  ES5 is def 
![download](https://user-images.githubusercontent.com/2019012/31359508-8386a9ae-ad07-11e7-81d9-89ecaa7e72cb.jpg) favorite.

Reply:
#### That was my joke...Nachos! 
![](https://media.giphy.com/media/s68kEBydcxoPK/giphy.gif)